sphinx6-upgrade.rst

```diff
@@ -48,9 +47,8 @@ Moving forwards,
 the number of upgrade paths for documentation projects
 are fewer and thus will lead to less choices for documentation owners and
 maintainers of Sphinx themes and extensions.
-At Read the Docs, we support both our main theme, `sphinx-rtd-theme`_, and a number of extensions for Sphinx.
-In light of that,
-we are direct beneficiaries of the reduced dependency matrix.
+This is a positive step forward for both documentation projects, themes and extensions.
+We recommend this upgrade in order to reduce complexities and future technical debt.
```

Comment:
This is a bit in conflict with the above line:

`we recommend that you do not upgrade to Sphinx 6 before you see announcements of Sphinx 6 support from these projects.`

We should probably clarify that language.

Reply:
```suggestion
If your Sphinx extensions and theme support Sphinx 6, we recommend this upgrade in order to reduce complexities and future technical debt.
```